incubator/codedx/codedx.props

```diff
@@ -0,0 +1,167 @@
+#   Copyright 2017 Code Dx, Inc
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+# NOTE: Any changes made to this config file will only take effect when
+# Tomcat is restarted.
+
+# db configuration for using MySQL
+swa.db.url = TPL_K8S_DB_URL
```

Comment:
Small note - the TPL_K8S_* prefix here is an attempt to inform the user that this symbol is specific to our chart, and not something the Code Dx itself will generate/use